input/pagecontent/reference.md

```diff
@@ -280,29 +362,27 @@ To set the top-level text of a CodeableConcept, the FSH expression is:
   * code.text = "Diagnosis of malignant neoplasm left breast."
   ```
     
-##### Quantity
+##### Representing `Quantity` Data Type
 
 FSH provides a shorthand that allows quantities with units of measure to be specified simultaneously, provided the units of measure are [Unified Code for Units of Measure](http://unitsofmeasure.org/) (UCUM) codes. The syntax is:
 
 ```
-* {Quantity type} = {number} '{valid UCUM unit}'
+* <Quantity> = {decimal} '{UCUM unit}'
 ```
 
 This syntax is borrowed from the [Clinical Quality Language](https://cql.hl7.org) (CQL).
 
 The value and units can also be set independently. To set the value of quantity value, the quantity `value` property can be set directly:
 
 ```
-* {Quantity type}.value = {number}
+* <Quantity>.value = {decimal}
 ```
 
-To set the units of measure independently, a Quantity can be bound to a value set or assigned a coded value. The FSH is:
+To set the units of measure independently of the value, a Quantity can be bound to a value set or assigned a coded value. The syntax is:
 
-```
-* {Quantity type} = {system}#{code} "{display text}"
-```
+<pre><code>* &lt;Quantity&gt; = {CodeSystem name|id|url}#{code} <i>"{display string}"</i></code></pre>
 
-Although it appears the quantity itself is being set to a coded value, this expression sets only the units of measure.
+Although it appears the quantity itself is being set to a coded value, this expression does in fact set the units of measure. This is a consequence of FHIR's definition of the Quantity data type.
```

Comment:
Clarified:

The `CodeSystem` corresponds to Quantity.system, `code` to Quantity.code, and display string to unit.

> **Note:** Although the previous example appears to set the Quantity itself to a coded value, this expression does in fact set the units of measure. This is a consequence of FHIR's definition of Quantity *as* a coded data type, rather than *having* a coded data type to represent the Quantity's units of measure.